backend/danswer/db/chat.py

```diff
@@ -250,6 +252,43 @@ def create_chat_session(
     return chat_session
 
 
+def duplicate_chat_session_for_user_from_slack(
+    db_session: Session,
```

Comment:
Comment for this function would be great- can foresee some confusion on this usage